ui/admin/tests/acceptance/targets/create-test.js

```diff
@@ -52,16 +53,26 @@ module('Acceptance | targets | create', function (hooks) {
     urls.targets = `${urls.projectScope}/targets`;
     urls.target = `${urls.targets}/${instances.target.id}`;
     urls.unknownTarget = `${urls.targets}/foo`;
-    urls.newTarget = `${urls.targets}/new`;
+    urls.newTCPTarget = `${urls.targets}/new?type=tcp`;
+    urls.newSSHTarget = `${urls.targets}/new?type=ssh`;
     // Generate resource couner
     getTargetCount = () => this.server.schema.targets.all().models.length;
     authenticateSession({});
   });
 
-  test('can create new targets', async function (assert) {
+  test('can create new targets of type TCP', async function (assert) {
     assert.expect(1);
     const count = getTargetCount();
-    await visit(urls.newTarget);
+    await visit(urls.newTCPTarget);
+    await fillIn('[name="name"]', 'random string');
+    await click('[type="submit"]');
+    assert.equal(getTargetCount(), count + 1);
+  });
+
+  test('can create new targets of type SSH', async function (assert) {
```

Comment:
SSH Target creation should be available JUST if the feature flag is enabled right? If so, how this test is passing without activating such feature?

Reply:
ssh-target feature flag is enabled for [dev env](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/config/environment.js#L138).

Reply:
do you mean in prod? 🤔

Reply:
Ahh ok that makes sense.

I was thinking on using this within the test: `enableFeature('ssh-target');` So the test does not depend on the feature being enabled.

But for now I think you are right, better keep it on the feature flag :)